include/nbl/asset/CCPUMeshPacker.h

```diff
@@ -71,38 +71,142 @@ class CCPUMeshPacker final : public IMeshPacker<ICPUMeshBuffer, MDIStructType>
 	using TriangleBatch = typename base_t::TriangleBatch;
 
 public:
-	CCPUMeshPacker(const SVertexInputParams& preDefinedLayout, const MeshPackerBase::AllocationParams& allocParams, uint16_t minTriangleCountPerMDIData = 256u, uint16_t maxTriangleCountPerMDIData = 1024u)
-		:IMeshPacker<ICPUMeshBuffer, MDIStructType>(preDefinedLayout, allocParams, minTriangleCountPerMDIData, maxTriangleCountPerMDIData)
-	{}
+	struct AllocationParams
```

Comment:
why aren't you inheriting from `AllocationParamsCommon` ?

we have a lot of duplicate code now.